lib/gollum/public/gollum/javascript/editor/gollum.editor.js

```diff
@@ -126,6 +126,10 @@
       }
         
       editor.setTheme("ace/theme/tomorrow");
+      editor.setOptions({
+        fontFamaly: 'Consolas, "Liberation Mono", Courier, monospace',
```

Comment:
Typo: `fontFamaly`

Reply:
you right, but it break fix... ))) need to dig more

Reply:
Thanks for looking in to it!